umn/source/_static/script.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python3
```

Comment:
usage of otcextensions and sdk is nice.
but do we have also possibility to descirbe the same procedure using CLI? If yes it would be good addition for customers who are not so skilled in python...

Reply:
Added overview section.